manifests/params.pp

```diff
@@ -58,95 +58,100 @@
   case $::osfamily {
     'Windows' : {
       # Windows prefixes normal paths with the Data Directory's path and leaves 'puppet' off the end
-      $dir_prefix                 = 'C:/ProgramData/PuppetLabs/puppet'
-      $dir                        = "${dir_prefix}/etc"
-      $codedir                    = "${dir_prefix}/etc"
-      $logdir                     = "${dir_prefix}/var/log"
-      $rundir                     = "${dir_prefix}/var/run"
-      $ssldir                     = "${dir_prefix}/etc/ssl"
-      $vardir                     = "${dir_prefix}/var"
-      $sharedir                   = "${dir_prefix}/share"
-      $bindir                     = "${dir_prefix}/bin"
-      $root_group                 = undef
-      $server_puppetserver_dir    = undef
-      $server_puppetserver_vardir = undef
-      $server_puppetserver_rundir = undef
-      $server_puppetserver_logdir = undef
-      $server_ruby_load_paths     = []
-      $server_jruby_gem_home      = undef
+      $dir_prefix                    = 'C:/ProgramData/PuppetLabs/puppet'
+      $dir                           = "${dir_prefix}/etc"
+      $codedir                       = "${dir_prefix}/etc"
+      $logdir                        = "${dir_prefix}/var/log"
+      $rundir                        = "${dir_prefix}/var/run"
+      $ssldir                        = "${dir_prefix}/etc/ssl"
+      $vardir                        = "${dir_prefix}/var"
+      $sharedir                      = "${dir_prefix}/share"
+      $bindir                        = "${dir_prefix}/bin"
+      $root_group                    = undef
+      $server_puppetserver_dir       = undef
+      $server_puppetserver_vardir    = undef
+      $server_puppetserver_rundir    = undef
+      $server_puppetserver_logdir    = undef
+      $server_puppetserver_logformat = undef
```

Comment:
Any reason not to default set it unconditionally to plain? You're skipping this on Windows anyway.

Reply:
Sure, will do.